src/components/ExtraAnnotations/index.tsx

```diff
@@ -54,14 +65,24 @@ const AssemblyExtraAnnotations: React.FC = () => {
         Header: 'Action',
         accessor: 'links.self',
         Cell: ({ cell }) => (
-          <a
-            href={cell.value}
-            className="vf-button vf-button--link"
-            style={{ whiteSpace: 'nowrap' }}
-            download
-          >
-            <span className="icon icon-common icon-download" /> Download
-          </a>
+          <>
+            {cell.row.original.attributes.description.label ===
+              'Analysis RO Crate' && (
+              <ROCrateBrowser
+                useButtonVariant
+                crateUrl={cell.value}
+                specificCrateFolder={`motus_${accession}`}
```

Comment:
This can be removed now

Reply:
@SandyRogers not sure what you mean here.
I think we still need the browser here. Could you explain more please ?

Reply:
Sure. I think we don't need to set the `specificCrateFolder` attribute since it is no longer used.

Reply:
@SandyRogers I've removed the reference